libs/sdk-ui-kit/src/Header/typings.ts

```diff
@@ -46,10 +46,12 @@ export interface IAppHeaderProps {
 
     userName: string;
 
+    expiredDate?: Date;
     disableHelpDropdown?: boolean;
     onHelpClick?: (isOpen: boolean) => void;
     helpMenuDropdownAlignPoints?: HelpMenuDropdownAlignPoints;
     showStaticHelpMenu?: boolean;
+    showCountdown?: boolean;
```

Comment:
Do we really need this prop? Can we rely on `expiredDate` only instead? I mean, we can pass a non-null value for `expiredDate` only when the expiration date should be displayed, can't we?